commands/modmail.py

```diff
@@ -35,10 +35,143 @@ async def cancel(self, button: discord.ui.Button, interaction: discord.Interacti
         self.value = False
         self.stop()
 
+class PersistentEvent(discord.ui.View):
+    def __init__(self, guild, bot):
+        super().__init__(timeout=None)
+        self.guild = guild
```

Comment:
```suggestion
```
We can bypass the need to store guild in the view since the interaction is initiated in the guild when they click the button.